galaxyproject/galaxy
# Tabular dataset display component

Screenshot:

![image](https://user-images.githubusercontent.com/155398/211218713-7926a6dc-eb02-443a-b69d-b3cddf79d4bd.png)

~~check replaceChildren tweak -- likely need to address other consumers, but this is a good change.~~

This initial implementation intentionally keeps the overly flexible row handling in place (variable column numbers, diverging from spec, basically rendering anything you throw at it).  We should follow up with more explicit handling for particular types when data can be assured to be clean -- this will allow for significant performance improvements.

## How to test the changes?
(Select all options that apply)
  1. Click eye for various tabular datasets, observe display.

## License
